### DISCUSSION TITLE
Somes suggestions (A dark mode in the ux)+(One notes the scrappy effect write everywhere)

### DISCUSSION THREAD

**Discussion by Embuscad2** [zim-desktop-wiki/zim-desktop-wiki#1920] [Ideas]
**Hello,**

I wonder if it would be possible to add to the Zim software, something that would allow it to have an all black user interface, a kind of dark mode that would be really great, or at least grey.

Please for my eyes :)

Another point, would it be possible to create the same thing as "One note" with the possibility to write text in the middle of the page everywhere you want and move the pieces of text.

I like the idea of not using copy/paste and just move the text, images, links, etc. to where I want them, it's a bit like managing your own tabs on a browser, it's practical.

There you go, and if not, thanks for your software, I'll suggest it to other people to whom I think it will be useful.

**Comment by dehesselle** [comment]
Dark mode is already possible, it's a matter of setting a GTK theme. See [here](https://unix.stackexchange.com/questions/14129/gtk-enable-set-dark-theme-on-a-per-application-basis) for example.

**Comment by jaap-karssenberg** [comment]
Hi,

The "one note model" has been discussed before here, but it simply is to
much effort to consider. Also it does not align naturally with storing data
in text files.

Regards,

Jaap
